addons/medical_treatment/functions/fnc_findMostEffectiveWound.sqf

```diff
@@ -40,33 +41,36 @@ private _effectivenessFound = -1;
     _x params ["_classID", "_partIndexN", "_amountOf", "_bleeding", "_damage"];
 
     // Ignore wounds on other bodyparts
-    if (_partIndexN == _partIndex) then {
-        private _woundEffectiveness = _effectiveness;
+    if (_partIndexN != _partIndexN) then { continue };
```

Comment:
Are these vars meant to be the exact same ones ("Am I not equal to myself ?") ? The latter one was gained from calling the script while the former is gained from the `forEach` iteration.